### ISSUE TITLE
Empty midi pattern makes it never send midi data

### ISSUE THREAD

**Issue by blueneogeo** [joslarson/react-bitwig#1]
If I have this code:

```
		<Midi
			label={props.label}
			pattern={{ status: 144 }}
			value={{ data1: NOTE_MIDI_MAP[props.index], data2: props.on ? 45 : 100 }}
			cacheOnInput
			cacheOnOutput
		/>
```

It will work well and send out midi signals when data1 or data2 changes. 

However if I move the status into the value and remove the pattern or have an empty pattern:

```
		<Midi
			label={props.label}
			value={{ status: 144, data1: NOTE_MIDI_MAP[props.index], data2: props.on ? 45 : 100 }}
			cacheOnInput
			cacheOnOutput
		/>
```

The midi component will never send out data. This may be by design (it needs a static pattern to know it's the same element?). However in my code I have to be able to change all three:

- status sets if the light is static, blinking or pulsing
- data1 sets the pad to update
- data2 sets the color of the pad

And my code needs to update all three.

Is this a bug or is there a different approach I should take?

Thank you for the nice library!

**Comment by blueneogeo** [comment]
I think I found a workaround, using two MIDI instances, one for each status, and only rendering the correct status one.

**Comment by joslarson** [comment]
@blueneogeo Thanks for pointing this out! Glad you found a work around, but you are right that this is a bug. Reopening to track this.

**Comment by joslarson** [comment]
@blueneogeo this should be fixed in v1.1.0 (just published)

**Comment by blueneogeo** [comment]
Sorry I saw this a bit late, but thank you for fixing!
